src/app/components/common/Footer.tsx

```diff
@@ -0,0 +1,58 @@
+import { ComponentProps } from 'react';
+
+interface FooterLinkProps extends Omit<ComponentProps<'a'>, 'className'> {
+  isExternal?: boolean;
+}
+
+function FooterLink({ isExternal = false, ...props }: FooterLinkProps) {
+  return (
+    <a
+      className="text-base-900 hover:text-base-600 dark:text-base-400 dark:hover:text-base-100 mr-auto transition"
+      target={isExternal ? '_blank' : undefined}
+      rel={isExternal ? 'noopener noreferrer' : undefined}
+      {...props}
+    />
+  );
+}
+
+export function Footer() {
+  return (
+    <div className="border-t-base-300 dark:border-t-base-600/60 mt-20 border-t">
+      <footer className="site-container m-auto w-full pb-6">
+        <div className="flex w-full flex-col justify-end gap-4 sm:flex-row sm:gap-10">
+          <div className="flex flex-col">
+            <h3 className="mt-6 mb-2 text-lg font-semibold">Sociálne siete</h3>
+            <FooterLink
+              href="https://www.instagram.com/"
+              isExternal
+            >
+              Instagram
+            </FooterLink>
+            <FooterLink
+              href="https://www.facebook.com/"
+              isExternal
+            >
+              Facebook
+            </FooterLink>
+          </div>
+          <div className="flex flex-col">
+            <h3 className="mt-6 mb-2 text-lg font-semibold">
+              Právne informácie a kontakt
+            </h3>
```

Comment:
PageSpeed Insights says:

>Heading elements are not in a sequentially-descending order
>
>Properly ordered headings that do not skip levels convey the semantic structure of the page, making it easier to navigate and understand when using assistive technologies. [Learn more about heading order](https://dequeuniversity.com/rules/axe/4.10/heading-order).

Should we perhaps use a different, non-heading element?